addons/repair/tests/test_repair.py

```diff
@@ -564,19 +567,28 @@ def test_repair_from_return(self):
         stock_return_picking = stock_return_picking_form.save()
         stock_return_picking.product_return_moves.quantity = 1.0
         stock_return_picking_action = stock_return_picking.create_returns()
-        return_picking = self.env['stock.picking'].browse(stock_return_picking_action['res_id'])
+        return_picking = self.env['stock.picking'].browse(stock_return_picking_action['res_id']).with_context(default_picking_type_id=stock_return_picking_action['context']['search_default_picking_type_id'])
         return_picking.action_set_quantities_to_reservation()
         return_picking.button_validate()
         self.assertEqual(return_picking.state, 'done')
 
         res_dict = return_picking.action_repair_return()
         repair_form = Form(self.env[(res_dict.get('res_model'))].with_context(res_dict['context']))
         repair_form.product_id = product
-        repair = repair_form.save()
-        repair.action_repair_start()
-        repair.action_repair_end()
-        self.assertEqual(repair.state, 'done')
-        self.assertEqual(len(return_picking.move_ids), 1)
+        try:
```

Comment:
Don't use try except in test. Instead use with self.assertRaise to test an exception